src/path_call_builder.js

```diff
@@ -0,0 +1,25 @@
+import {CallBuilder} from './call_builder';
+
+/**
+* @class PathCallBuilder
+*/
+export class PathCallBuilder extends CallBuilder {
+    /*
+    * @constructor
+    */
+    constructor(url, source, destination, destination_type, destination_amount) {
+        super(url);
+        this.url.segment('paths');
+        this.url.addQuery('destination_account', destination);
+        this.url.addQuery('source_account', source);
+        this.url.addQuery('destination_amount', destination_amount);
+
+        if (!destination_type.isNative()) {
+            this.url.addQuery('destination_asset_type', 'credit_alphanum4');
```

Comment:
You're right. Can you create PR in js-stellar-base from your branch? I will merge it.